indico/util/mdx_latex_test.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from indico.util import mdx_latex
+from markdown import Markdown
+import pytest
+
+
+_md = Markdown(safe_mode='remove')
+_latex_md = mdx_latex.LaTeXExtension()
+_latex_md.extendMarkdown(_md, _md.__dict__)
+
+
+@pytest.mark.parametrize(('input', 'expected'), [
+    ('\nplain text\n', 'plain text'),
+    ('plain text with reserved characters &^{}', r'plain text with reserved characters \&\^{}\{\}'),
+    ('**bold text**', r'\textbf{bold text}'),
+    ('**bold with &reserved{} characters**', r'\textbf{bold with \&reserved\{\} characters}'),
+    ('*italic text*', r'\emph{italic text}'),
+    ('*italic text with reserved chars &{}*', r'\emph{italic text with reserved chars \&\{\}}'),
+    ('[link with reserved chars&{}](www.indico.github.io)', r'\href{www.indico.github.io}{link with reserved chars\&\{\}}'),
```

Comment:
I guess you mean `http://indico...`. I know it's test data, but better if it looks good :wink: